token-service/src/main/java/org/p2p/token/service/repository/configurator/CanBeHiddenConfigurator.kt

```diff
@@ -0,0 +1,15 @@
+package org.p2p.token.service.repository.configurator
+
+import org.p2p.core.token.MetadataExtension
+import org.p2p.core.token.Token
+import org.p2p.core.token.TokenExtensions
+
+class CanBeHiddenConfigurator(
+    private val extensions: MetadataExtension,
+    private val tokenExtensions: TokenExtensions
```

Comment:
i would not pass tokenExtensions here as a field, take tokenExtensions from token

```
 override fun config(token: Token.Active): TokenExtensions {
        val tokenExtensions = token.tokenExtensions
        return tokenExtensions.copy(canTokenBeHidden = extensions.canBeHidden ?: true)
    }
```